github/copilot.go

```diff
@@ -186,7 +185,7 @@ func (s *CopilotService) ListCopilotSeats(ctx context.Context, org string, opts
 // AddCopilotTeams adds teams to the Copilot for Business subscription for an organization.
 //
 // GitHub API docs: https://docs.github.com/en/rest/copilot/copilot-for-business#add-teams-to-the-copilot-for-business-subscription-for-an-organization
-func (s *CopilotService) AddCopilotTeams(ctx context.Context, org string, teamNames []string) (*SeatAssignments, *Response, error) {
+func (s *CopilotService) AddCopilotTeams(ctx context.Context, org string, teamNames *SelectedTeams) (*SeatAssignments, *Response, error) {
```

Comment:
Sorry, that's not what I meant... this should still be `teamNames []string` and then an anonymous struct should be used _within_ this method to create the _actual_ struct being sent.  See above.